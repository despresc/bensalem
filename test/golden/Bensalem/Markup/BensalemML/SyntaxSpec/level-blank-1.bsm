\#levtag

  \&layout
    lc

 \#levtag2
