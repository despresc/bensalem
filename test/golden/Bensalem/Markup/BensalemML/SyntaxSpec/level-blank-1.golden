fromList
 [ ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 0
     , srcLine = 1
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 8
     , srcLine = 1
     , srcCol = 9
     }
    }
   , elementArgPresentation = PresentLevel
   , elementName = "levtag"
   , elementAttrs = fromList []
   , elementArg = fromList
    [ LineEnd
    , LineEnd
    , LineSpace 2
    , ElementNode
     ( Element
      { elementTagPos = SrcSpan
       { srcSpanName = ""
       , srcSpanStart = SrcPos
        { srcOffset = 12
        , srcLine = 3
        , srcCol = 3
        }
       , srcSpanEnd = SrcPos
        { srcOffset = 20
        , srcLine = 3
        , srcCol = 11
        }
       }
      , elementArgPresentation = PresentLayout
      , elementName = "layout"
      , elementAttrs = fromList []
      , elementArg = fromList
       [ LineEnd
       , LineSpace 4
       , PlainText "lc"
       ]
      }
     )
    , LineEnd
    , LineEnd
    , LineSpace 1
    ]
   }
  )
 , ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 30
     , srcLine = 6
     , srcCol = 2
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 39
     , srcLine = 6
     , srcCol = 11
     }
    }
   , elementArgPresentation = PresentLevel
   , elementName = "levtag2"
   , elementAttrs = fromList []
   , elementArg = fromList [ LineEnd ]
   }
  )
 ]